contents/blog/how-to-fix-your-growth-team.md

```diff
@@ -0,0 +1,136 @@
+
+You keep trying different experiments, but none have a meaningful impact and become a waste of time. It feels like you're always picking the wrong projects to work on.
+
+Alexey says the cause of this is teams not spending enough time estimating the effort and impact of their projects. 
+
+"Typically, teams prioritize their roadmaps by using frameworks such as [RICE](https://www.intercom.com/blog/rice-simple-prioritization-for-product-managers/) and assigning t-shirt size estimates ("S", "M", "L"). This doesn't go deep enough. Spending only five minutes estimating the impact from something your team will spend weeks on doesn't make sense."
+
+### How to fix it 
+
+Once you have shortlisted the ideas you want to work on, the next step is to **estimate the dollar impact of each one**.  
+
+Then, use this dollar amount together with your effort estimates to calculate a prioritization score:
+
+![How to calculate your prioritization score](../images/blog/how-to-fix-your-growth-team/how-to-calcluate-prioritization-score.png)
+
+Because you'll have a clearer idea of what to expect from your experiments, you'll prioritize the ones more likely to win. The table below shows an example:
+
+![Example of prioritized projects table](../images/blog/how-to-fix-your-growth-team/prioritization-table.png)
+
+Alexey says that **one of their biggest wins came from following this approach**:
+
+"At MasterClass, we had a single subscription type. We knew we would eventually need to add tiered pricing, but this would be a huge, months-long company effort. We had no idea how much this would impact our revenue, so we struggled to prioritize it.
+
+I knew we needed a dollar estimate, so we created a simple [fake door test](/tutorials/fake-door-test). Users could select their preferred tier – but at checkout, we let them know they were being upgraded to the best tier and at the cheapest price. In reality, this was the existing one-size-fits-all pricing, and we hadn't actually built any of the functionality to support the new tiers.
+
+This saved us months of engineering time and enabled us to understand how much we could gain if we built this. Once we knew how much money the company stood to make, we prioritized it immediately! And it was incredibly easy to ask the rest of the company to work on it for a few months."
+
+## Problem 3: You're running out of experiment ideas
+
+Your growth team has been around for a few quarters and you've picked all the low-hanging fruit. Your new experiments now require more effort AND have less impact. Your momentum is stalling and your team is beginning to feel demotivated.
+
+### How to fix it
+
+[John Egan](https://jwegan.com/growth-hacking/managing-growth-teams-portfolio-step-step-guide/), former head of growth engineering at Pinterest, recommends two levers you can pull:
+
+1. Make it everyone's job to come up with project ideas.
+2. Make your engineers own their experiments.
```

Comment:
Potentially don't need these if you're going to use them as titles directly after?